keyboards/xd75/keymaps/jarred/config.h

```diff
@@ -0,0 +1,8 @@
+#ifndef CONFIG_USER_H
```

Comment:
If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

Reply:
@noroadsleft says "Delete this file, blah blah blah, I've heard it a hundred times already."

:grinning: